PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -278,17 +278,22 @@ def miniAOD_customizeMC(process):
     process.photonMatch.src = cms.InputTag("reducedEgamma","reducedGedPhotons")
     process.tauMatch.matched = "prunedGenParticles"
     process.tauGenJets.GenParticles = "prunedGenParticles"
+    process.patJetPartons.particles = "prunedGenParticles"
     process.patJetPartonMatch.matched = "prunedGenParticles"
     process.patJetPartonMatch.mcStatus = [ 3, 23 ]
     process.patJetGenJetMatch.matched = "slimmedGenJets"
+    process.patJetPartonsAK8.particles = "prunedGenParticles"
```

Comment:
Only one instance of the `HadronAndPartonSelector` per job should be sufficient. So no need to introduce `process.patJetPartonsAK8` in addition to `process.patJetPartons`. I guess this is related with `postfix = 'AK8', which I don't think is needed.